schemas/answ_action_response.fbs

```diff
@@ -0,0 +1,17 @@
+// A message for holding information on the (immediate) outcome of a command to (or state change of) a filewriter
+
+file_identifier "answ";
+
+enum ActionType : byte { StartJob = 0, SetStopTime, StopNow, HasStopped }
+enum ActionOutcome : byte { Success = 0, Failure }
+
+table ActionResponse {
+    service_id : string (required);       // The identifier for the instance of the file-writer that responded to the command
+    job_id : string (required);           // A unique identifier for the file writing job
+    action : ActionType;                  // What type of command is this a response to?
+    outcome : ActionOutcome;              // Was the command accepted (action successfull)?
+    message : string;                     // Must hold an error message if outcome was "Failure", is optional otherwise.
+    command_id : string;                  // Holds an id that is unique to the command that this is the response to. In the case of start commands, the id is the job-id. In the case of an "has stopped message", this field may be ignored. 
```

Comment:
If I send two start commands with the same job_id in quick succession, should the first succeed and and the second fail? Could using the job_id as the command_id cause a problem with that?

Reply:
1. This is a good question that is impossible to answer due to how we are implementing the file-writer pool functionality.
2. Using the job-id as the command-id will not change the outcome of if a file-writing job is started as such. Rather this is file-writer implementation specific.

Reply:
I think we should define what should happen when a second stop command comes in, though. 
"while it can" the file writer should try to honour the latest request for a later or earlier end. But once it started winding down, I think re-starting should fail.

Reply:
@zjttoefs The handling of stop commands is currently exactly as you request it.